tests/test_spend.py

```diff
@@ -171,7 +171,16 @@ def sign_and_broadcast(psbt):
     psbt = PSBT.from_base64(res["psbt"])
     sign_and_broadcast(psbt)
     assert len(psbt.o) == 4
-    assert len(res["warnings"]) == 0
+    assert bitcoind.rpc.getmempoolentry(deposit_d)["ancestorsize"] == 165
+    assert bitcoind.rpc.getmempoolentry(deposit_d)["fees"]["ancestor"] * COIN == 165
+    # ancestor vsize at feerate 2 sat/vb = ancestor_fee / 2 = 165 / 2 = 82
+    # extra_weight <= (extra vsize * witness factor) = (165 - 82) * 4 = 332
+    # additional fee at 2 sat/vb (0.5 sat/wu) = 332 * 0.5 = 166
```

Comment:
Wouldn't it be even more obvious (and robust) to perform this calculation directly in Python?

Reply:
Oh yeah, I didn't think of that :) Will bear that in mind if coming back to this code.

Reply:
Adapting this to Taproot i basically translated this to Python.